galasa-extensions-parent/dev.galasa.ras.couchdb/src/main/java/dev/galasa/ras/couchdb/internal/CouchdbValidatorImpl.java

```diff
@@ -424,7 +426,7 @@ private void checkIndex(CloseableHttpClient httpClient, URI rasUri , int attempt
                     // Someone possibly updated
                     attempts++;
```

Comment:
This max attempt count, is it the same or different to the Conflict one above in this PR ?